src/GraphControl/Control/Grapher.cpp

```diff
@@ -389,4 +434,98 @@ namespace GraphControl
             e->Handled = true;
         }
     }
+
+    void Grapher::OnPointerWheelChanged(PointerRoutedEventArgs^ args)
+    {
+        PointerPoint^ currentPointer = args->GetCurrentPoint(/*relative to*/ this);
+
+        double delta = currentPointer->Properties->MouseWheelDelta;
+
+        // The maximum delta is 120 according to:
+        // https://docs.microsoft.com/en-us/uwp/api/windows.ui.input.pointerpointproperties.mousewheeldelta#Windows_UI_Input_PointerPointProperties_MouseWheelDelta
+        // Apply a dampening effect so that small mouse movements have a smoother zoom.
+        constexpr double scrollDamper = 0.15;
```

Comment:
Curious as to when constexpr makes sense within functions instead of above where the other constexpr are defined

Reply:
Yes, constexpr makes just as much sense here.  The compiler will still perform [constant initialization](https://en.cppreference.com/w/cpp/language/constant_initialization) of the variable which, in this case, means placing a definition in the program's .rdata section.  We should also prefer having the constant here instead of at the top, because of http://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#Res-introduce